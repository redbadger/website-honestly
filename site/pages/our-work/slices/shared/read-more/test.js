// @flow

import React from 'react';
import { shallow } from 'enzyme';

import ReadMore from '.';

describe('site/pages/our-work/slices/shared/read-more', () => {
  it('renders the correct class by default', () => {
    const component = shallow(<ReadMore />);
    expect(component).toMatchSnapshot();
  });

  it('renders the centered class', () => {
    const component = shallow(<ReadMore center />);
    expect(component).toMatchSnapshot();
  });
});
