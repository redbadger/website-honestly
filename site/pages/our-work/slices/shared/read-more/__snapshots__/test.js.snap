// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`site/pages/our-work/slices/shared/read-more renders the centered class 1`] = `
<div
  className="linksCenter"
>
  <p
    className="readmore"
  >
    Read more
  </p>
</div>
`;

exports[`site/pages/our-work/slices/shared/read-more renders the correct class by default 1`] = `
<div
  className="links"
>
  <p
    className="readmore"
  >
    Read more
  </p>
</div>
`;
